docs/llvm/overview/index.md

```diff
@@ -0,0 +1,292 @@
+# Getting Started
+
+In this section, we will briefly describe the basics and concepts related to compilers and introduce the LLVM project.
+
+> This section assumes that you do not have any background in compilers and that you already have an understanding of programming and can write simple C code.
+
+## What is a compiler
+
+Conceptually, a compiler is a **computer program** that converts a programming language into another language. What we usually call a compiler generally converts a higher-level programming language such as C, C++, Fortran, etc. into a lower-level assembly language or machine code.
+
+**Note that a compiler is not an editor, much less an IDE.**
```

Comment:
```suggestion
**Note that a compiler is neither an editor nor an IDE.**
```

Reply:
I think the original sentence actually makes more sense?